packages/data-context/src/sources/FileDataSource.ts

```diff
@@ -9,6 +9,10 @@ import { toPosix } from '../util/file'
 
 const debug = Debug('cypress:data-context:sources:FileDataSource')
 
+export const matchGlobs = async (globs: string | string[], globbyOptions?: GlobbyOptions) => {
+  return await globby(globs, globbyOptions)
+}
```

Comment:
Our pinned version of globby exports the [function itself](https://github.com/sindresorhus/globby/blob/v11.1.0/index.js#L114) as the module export, making it difficult to stub calls to it for unit testing purposes. 

So I added a little wrapper here so I could stub these calls appropriately, but I can remove this if I'm missing an obvious way to stub the module directly.